contrib/examples/actions/workflows/mistral-env-var.yaml

```diff
@@ -0,0 +1,14 @@
+version: '2.0'
+
+examples.mistral-env-var:
+    description: A basic workflow that illustrates how to get the workflow's env vars.
+    type: direct
+    output:
+        stdout: <% $.stdout %>
+    tasks:
+        task1:
+            action: core.local
+            input:
+                cmd: echo <% env().st2_execution_id %>
```

Comment:
I suggest we show the example with the URL. It's a real use case people are asking and we use broadly. Hardcoding IP is fine, they'll get it. Like `https://172.168.50.11/#/history/<% env().st2_execution_id %>/general`

This gives an idea, and answers a bunch of know-how.